tests/helpers/steps.ts

```diff
@@ -0,0 +1,102 @@
+import { Page } from '@playwright/test';
+
+export async function navigateHomePage(page: Page, specificPath?: string) {
+  await page.goto('/');
+  if (specificPath === 'energy-calculator') {
+    await page.goto('/co_energy_calculator/landing-page');
+  }
+}
```

Comment:
A nitpick on this. The `navigateHomePage` can be made more reusable, like instead of checking the specific path through a conditional we can pass it in the URL if a value is given. 
```suggestion
export async function navigateHomePage(page: Page, specificPath?: string) {
  await page.goto('/');
  if (specificPath) {
    await page.goto(specificPath);
  }
}
```

Reply:
done